stories/assets/js/Tab_custom.js

```diff
@@ -1,16 +1,6 @@
 /* Tab JS start custom */
```

Comment:
@harishkumar-srijan Change the existing comments to tabs.js library here.
Eg. 
```
/**
 * tabs.js library
 */

```

Reply:
done